torch/testing/_internal/common_methods_invocations.py

```diff
@@ -6672,12 +6709,14 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
     OpInfo('ravel',
            dtypes=all_types_and_complex_and(torch.bool, torch.float16, torch.bfloat16),
            supports_out=False,
+           supports_forward_ad=True,
            sample_inputs_func=sample_inputs_ravel,
            ),
     OpInfo('reshape',
            dtypes=all_types_and_complex_and(torch.bool, torch.float16, torch.bfloat16),
            sample_inputs_func=sample_inputs_view_reshape,
            supports_out=False,
+           supports_forward_ad=True,
```

Comment:
Does this opinfo only have contiguous inputs? If so, we should add some non-contiguous inputs to really check that reshape works.

https://github.com/pytorch/pytorch/blob/842a831f53bc9305c0553f3b2096afc6c1c6efe4/torch/testing/_internal/common_methods_invocations.py#L3954-L3968

Reply:
It wasn't indeed.
Updated,